tests/templates/kuttl/authentication/create-trino-cluster.yaml.j2

```diff
@@ -17,7 +17,7 @@ metadata:
   namespace: $NAMESPACE
 spec:
   image:
-    productVersion: "{{ test_scenario['values']['trino-latest'] }}"
+    productVersion: "'"{{ test_scenario['values']['trino-latest'] }}"'"
```

Comment:
This generates fields like this:
```
productVersion: "'"414"'" # double-quote - quote - double-quote 4 1 4 ...
```
Is this intentional ?

Reply:
We actually need this change to fix the kuttl nightly tests. As this runs through an "echo" statement we need to quote it twice, as bash removes the outer quotes